src/uct/base/uct_worker.c

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
```

Comment:
you moved the code without preserving copyrights. Please copy-n-paste copyright from the original file